pynest/nest/lib/hl_api_simulation.py

```diff
@@ -460,3 +468,109 @@ def DisableStructuralPlasticity():
 
     """
     sr('DisableStructuralPlasticity')
+
+@check_stack
+def Dump(selections: List = ("nodes", "synapses")) -> Dict:
+    """Returns a dictionary containing the current net in memory for serialization.
+
+    Only a subset of the data can be serialized.
+
+
+    Parameters
+    ----------
+    selections : List[strings], optional
+        Only obtain a subset of the net parameters. Options are "nodes" and/or "synapses"
+
+
+    Returns
+    -------
+    Dict:
+        Object containing the specified serialization sorted by dict key likes "nodes" or "synapses"
+
+
+    See Also
+    ---------
+
+    Load : Function to construct the net from the data.
+
+    """
+
+    sli_func("({}) M_WARNING message".format("Dumping is currently only supported for a single process."))
```

Comment:
I don't think that warning is a good idea : if it does not work it should crash and burn in a very explicit way ;)

Reply:
I think if you run it with MPI enabled on several machines it won't crash but just modify the local machine. Do you think it should check for that case explicitly?